### ISSUE TITLE
IDE sometimes permanently loop in refresh till restart

### ISSUE THREAD

**Issue by Nightenom** [neoforged/NeoGradle#44]
Occasionally vscode enters into permanent refresh loop because of file watcher for `build/neoForm/neoFormJoined<version>/unpacked/config/inject/mcp/client/Start.java`
Reloading the window has ~50 % chance to stop it, always happens after clean import, during dev some actions may start the loop too (probably something what triggers NG to do neoform setup?)
[Trace log of vscode java extension messages](https://gist.github.com/Nightenom/0bdfdb79083a263928c26ea8a7e16269)

**Comment by marchermans** [comment]
I can not reproduce this in the current version, can you check?

**Comment by Anonymous2416** [comment]
same issue for me 

https://github.com/neoforged/NeoGradle/assets/99367774/d1f070ab-69bc-4f7f-8a13-bc8cd0bb5a14

**Comment by Nightenom** [comment]
mitigation in `.vscode/settings.json`: 
```json
"files.watcherExclude": {
    "**/build/neoForm/**": true,
}
```

issue:
everything under `build/neoform/<version>/patches` gets notified in cycle of `neoFormPatch` task, example log entry:
```
2024-06-26 12:58:11.609 [trace] [File Watcher (parcel)]  >> normalized [CHANGED] p:\data\github\BlockUI\build\neoForm\1.20.4-20231207.154220\patches\server\net\minecraft\world\level\block\BaseEntityBlock.java.patch
```

**Comment by Nightenom** [comment]
Cannot replicate after latest ng changes, probably solved in #232, feel free to reopen if someone is still affected
